### PR TITLE
Feature/support for errbit

### PR DESCRIPTION
**Pull request by argos83**
Support to API compatible Airbrake alternatives (e.g. Errbit)

### CODE REVIEW

File: airbrake/__about__.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 __summary__ = 'Python SDK for airbrake.io'
 __author__ = 'BK Box, Sam Stavinoha'
 __email__ = 'samuel.stavinoha@rackspace.com'
-__version__ = '1.2.1'
+__version__ = '1.3.0'
```
Comment (argos83) [inline airbrake/__about__.py]: based on: http://semver.org/  (added backwards compatible functionality)

File: pylintrc [hunk 2]
```diff
@@ -12,7 +12,7 @@ good-names=f,e,i,j,k,v,ex,_,x,y,z,kw
 # No camel case globals, max length 30
 const-rgx=(^([A-Za-z_](?!(([A-Z][a-z0-9]+)+))){2,30}$)
 max-args = 7
-max-attributes = 8
+max-attributes = 9
```
Comment (argos83) [inline pylintrc]: Sorry that I just increased this, but I wanted to introduce the changes in the least intrusive way I could, and keeping the max-attribute at 8 would have required some refactoring in notifier.py not related to the change proposed in this pull request.

**Comment by samstav** [comment]
I think the project at `circleci.com/gh/airbrake/airbrake-python` got disabled? Could one of the owners on this repo go here and follow the project?

https://circleci.com/gh/airbrake/airbrake-python

/cc @phumpal @vmihailenco @kyrylo

File: pylintrc (hunk 2 above)
Comment (samstav) [inline pylintrc]: :+1:

**Comment by argos83** [comment]
Now it passes on circleci: https://circleci.com/gh/argos83/airbrake-python/8
There was a previous PEP8 non-compliance plus a few others that I introduced, all of them are fixed now.

File: README.md [hunk 3]
```diff
@@ -46,6 +46,26 @@ except Exception:
     logger.exception("Bad math.")
 ```
 
+###setup for Airbrake alternatives (e.g. Errbit)
```
Comment (phumpal) [inline README.md]: @joegod - perhaps the wording `Airbrake alternatives` needs to be changed. Maybe

- Airbrake compatible backends
- Airbrake compliant backends

Just a thought.

File: README.md [hunk 4]
```diff
@@ -46,6 +46,26 @@ except Exception:
...
+
+There are self-hosted alternatives to Airbrake which provide a compatible API (e.g. [Errbit](https://github.com/errbit/errbit)).
```
Comment (phumpal) [inline README.md]: @joegod - Another suggestion - maybe something about our onpremise offering?

"*Airbrake [Enteprise](https://airbrake.io/enterprise) and self-hosted alterantives, such as [Errbit](https://github.com/errbit/errbit), provide a compatible API.*"

File: airbrake/__about__.py (hunk 1 above)
Comment (phumpal) [inline airbrake/__about__.py]: :+1:

File: pylintrc (hunk 2 above)
Comment (phumpal) [inline pylintrc]: Makes sense :+1:

**Comment by kyrylo** [comment]
@samstav I just tweaked some settings and it *seems* to be building now. I cannot rebuild this PR, though, because Circle CI doesn't see this branch.

**Comment by argos83** [comment]
@kyrylo, if it helps: here's the latest CircleCI build from my fork:
https://circleci.com/gh/argos83/airbrake-python/9

**Comment by samstav** [comment]
LGTM, ok to merge? 

/cc @kyrylo @phumpal

**Comment by phumpal** [comment]
Circle tests are passing. Thanks for fixing that @kyrylo 

@argos83 disregard my previous comments regarding README updats.

@samstav LGTM :+1:

**Comment by argos83** [comment]
@phumpal, @samstav. Can you have the latest version published in pypi?. `pip install -U airbrake` still gets 1.2.1.
Thanks!

**Comment by phumpal** [comment]
@samstav I don't think we have access to the airbrake package on pypi. Do you mind taking care of this?

I'll shoot you an email with our user details. 

Thanks!

**Pull request merged**